aws_s3_bucket.tf

```diff
@@ -10,6 +10,10 @@ resource "aws_s3_bucket" "noS3BucketSseRules" {
     Name        = "nos3BucketSseRules"
     Environment = "Dev"
   }
+
+  versioning {
```

Comment:
**Accurics remediation**
 
**[HIGH]: S3 bucket accessible to all authenticated users.**
 
**Vulnerability**
S3 bucket Access is allowed to all AWS Account Users.
 
**Remediation**
S3 bucket Access should be limited to specific users and roles.

source: accurics-